register.js

```diff
@@ -0,0 +1,54 @@
+
+
+document.getElementById('submitButton').addEventListener('click', (e) => {
+    //Prevents default form action from happening
+    e.preventDefault();
+
+    //Get Previous Items as array
+    let previousClients = getLocalStorage()
+    
+    console.log(previousClients, "Previous Clients")
+
+    //Get Input Values
+    let nameValue = document.querySelector('input[type=text]').value;
+    let emailValue = document.querySelector('input[type=mail]').value;
+    let phoneValue = document.querySelector('input[type=tel]').value;
```

Comment:
Why did you use querySelector, and what are the other options ?

Reply:
i can get the value by id , name, and tag name.

i can add an id attribute and use >> var number=document.getElementById("number").value;

Reply:
Yeah cool, but what made you use querySelector